src/test/client/ops.cc

```diff
@@ -0,0 +1,35 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+/*
+ * Ceph - scalable distributed file system
+ *
+ * Copyright (C) 2022 New Dream Network
+ *
+ * This is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License version 2.1, as published by the Free Software
+ * Foundation.  See file COPYING.
+ *
+ */
+
+#include <iostream>
+#include <errno.h>
+#include "gtest/gtest.h"
+#include "TestClient.h"
+
+TEST_F(TestClient, CheckDummyOP) {
+  int res = client->check_dummy_op(myperm);
+  ASSERT_EQ(res, -EOPNOTSUPP);
+}
+
+TEST_F(TestClient, CheckClientEviction) {
+  int res = client->check_client_eviction();
+  ASSERT_EQ(res, -EINVAL);
+}
+
+TEST_F(TestClient, CheckUnknownReclaimFlag) {
+  int res = client->check_unknown_reclaim_flag();
+  ASSERT_EQ(res, (-EINVAL || -EOPNOTSUPP));
```

Comment:
Will this work ?

Maybe you should use `EXPECT_THAT(res, Eq(Gt(-EINVAL), Eq(-EOPNOTSUPP)));`

More detail please refer to https://github.com/ceph/ceph/pull/47063/commits/91fcdbb58e572a5e891e986b85d70cd84f3daf7d

Reply:
I'll run it locally, need to build cluster again, let's see how it goes

Reply:
I'm making change here as creating a fresh build is gonna take sometime as some ninja files went missing.

Reply:
@lxbsz maybe it should be `EXPECT_THAT(res, AnyOf(Eq(-EOPNOTSUPP), Eq(-EINVAL)));`?

Reply:
Yeah, right.

Reply:
done, thanks for the tip